app/views/admin/loans/_form.html.slim

```diff
@@ -38,11 +39,11 @@
     = f.input_field :status_value, collection: Loan.status_options, include_blank: true
 
   = f.input :amount
-    .view-element = format_currency(@loan.amount, @loan.currency, tooltip: false)
+    .view-element = @loan.id ? format_currency(@loan.amount, @loan.currency, tooltip: false) : ''
```

Comment:
The new loan form was broken here and in two other spots. If a loan does not exist yet, then these places where loan.id is used would not get rendered, which ensures the new loan form can load.